package.json

```diff
@@ -117,5 +117,8 @@
   },
   "alias": {
     "yaml": "yaml/browser/dist/index.js"
+  },
+  "resolutions": {
+    "htmlnano": "2.0.3"
```

Comment:
Does running `yarn` locally give some changes to the yarn.lock we should add too?

Reply:
Updated `yarn.lock`. To the final testing now.